frontend/src/components/SettingsGeneral.vue

```diff
@@ -95,6 +99,21 @@ const timezones = Intl.supportedValuesOf('timeZone');
   </div>
   <div class="pl-6">
     <div class="text-xl">{{ t('heading.dateAndTimeFormatting') }}</div>
+    <div class="mt-6 pl-6">
+      <div class="text-lg">{{ t('label.startOfTheWeek') }}</div>
+      <label class="mt-4 flex items-center pl-4">
+        <div class="w-full max-w-2xs">{{ t('label.startOfTheWeek') }}</div>
+        <select
+          v-model="user.data.settings.startOfWeek"
+          class="w-full max-w-sm rounded-md"
+          data-testid="settings-general-star-of-week-select"
```

Comment:
typo, missing the t in settings-general-`star`-of-week-select

Reply:
Good catch, fixed. Thanks!